tree/ntuple/test/ntuple_type_name.cxx

```diff
@@ -216,13 +216,13 @@ TEST(RNTuple, TypeNameTemplatesNestedAlias)
    ASSERT_EQ(2, hashSubfields.size());
    EXPECT_EQ("fHash", hashSubfields[0]->GetFieldName());
    EXPECT_EQ("std::string", hashSubfields[0]->GetTypeName());
-   EXPECT_EQ("EdmHash<1>::value_type", hashSubfields[0]->GetTypeAlias());
+   EXPECT_EQ("EdmHash::value_type", hashSubfields[0]->GetTypeAlias());
```

Comment:
This change is incorrect: `EdmHash` is a template class and requires template arguments. Clang stores the inner class outside the template instantiation as an optimization, but the type must have arguments.